.github/workflows/reusable-build.yml

```diff
@@ -34,6 +34,15 @@ jobs:
           - base
           - lazurite
           - vauxite
+        include:
+          - image_name: silverblue
+            base_image: quay.io/fedora/fedora-silverblue
+          - image_name: kinoite
+            base_image: quay.io/fedora/fedora-kinoite
+          - image_name: sericea
+            base_image: quay.io/fedora/fedora-sway-atomic
+          - image_name: onyx
+            base_image: quay.io/fedora/fedora-budgie-atomic
```

Comment:
Should we just go with the upstream names so they end up being `sway-main` and `budgie-main`?

Reply:
We can do, though it may be best waiting for the next major version rather than introducing breaking changes in the middle of a cycle